apps/game_backend/priv/curse_of_mirra/quests_descriptions.json

```diff
@@ -0,0 +1,45 @@
+[
+    {
+        "config_id": 1,
+        "description": "Win three matches with Muflus",
+        "type": "daily",
+        "objective": {
+            "field": "result",
+            "value": 3,
+            "comparison": "greater",
+            "scope": "match"
+        },
+        "conditions": [
```

Comment:
Can you explain a bit on the distinction between `objective` and `conditions`? Like why can't we have the `objective` as one more condition?

Reply:
Sure! the objective will determine the amount of progress you need to have in order to consider a quest completed, the conditions are a set of comparators to see if a match result is a valid candidate to contribute to quest completion

> Like why can't we have the objective as one more condition?
About this, how would define progression in a quest if everything is an objective? this was the solution i came up with but im open to ideas